sycl/doc/SpecializationConstants.md

```diff
@@ -167,27 +167,27 @@ Plugin Interface function for each entry, taking the `spec_id` from the ID map.
 
 Say, the POD type is
 
-```
+```c++
 struct A {
   int x;
   float y;
 };
 
 struct POD {
   A a[2];
-  int b;
+  cl::sycl::vec<int, 2> b;
```

Comment:
This example might actually be confusing, because `cl::sycl::vec` is not a POD-type in our implementation, because it is not trivially copyable due to the fact that there is non-trivial copy constructor (I don't know why it is coded in this way, probably @romanovvlad can say something about this).

I guess that from user point of view it would be nice to have possibility to use vectors as specialization constants, so, what should I do here: do I put a vector into the example or do I leave it as-is?